src/arm/96boards.c

```diff
@@ -64,7 +64,8 @@ int db410c_chardev_map[MRAA_96BOARDS_LS_GPIO_COUNT][2] = {
 };
 
 const char* db410c_serialdev[MRAA_96BOARDS_LS_UART_COUNT] = { "/dev/ttyMSM0", "/dev/ttyMSM1" };
-const char* db410c_led[MRAA_96BOARDS_LED_COUNT] = { "user1", "user2", "user3", "user4", "bt", "wlan" };
+const char* db410c_led[MRAA_96BOARDS_LED_COUNT] = { "user1", "user2", "user3", "user4", "bt", "wla"
+                                                                                              "n" };
```

Comment:
You should wrap the entire string to next line. I know it is coming from clang-format but it doesn't look good.

Reply:
Yeah, `clang-format` is way too "smart" here :rofl: